Clarification: MIN|MAX/@value for arrays are usually single literals.

msdemlei opened the pull request:
The PR also starts the WD cycle for 1.5.

msdemlei commented:
On Thu, Jan 19, 2023 at 05:22:43AM -0800, Marco Molinaro wrote:
> I wonder if an example below the added text would be useful or commonplace.

I think you'd have my vote.  I was also thinking about
naming-and-shaming the max thing for SODA polygons...

tomdonaldson reviewed (changes requested):
I agree with @molinaro-m that examples would be very helpful here.  Illustrating each of array, non-array, and xtype would make the text easier to digest.

I think it's fine if the examples follow on in another PR.  Perhaps if we file an issue to that effect we won't forget, or can discuss its priority further.

msdemlei commented:
On Wed, Jan 25, 2023 at 10:47:20AM -0800, Tom wrote:
> @tomdonaldson requested changes on this pull request.
> 
> I agree with @molinaro-m that examples would be very helpful here.
> Illustrating each of array, non-array, and xtype would make the
> text easier to digest.

Hm... we *could* say we want to do a better job here than in other
places, but frankly, if I made a priority list of where the spec
should add examples, this one would not be in the top 10 :-)

> +\previousversion[http://www.ivoa.net/documents/VOTable/20191021/]
> +                {http://www.ivoa.net/documents/VOTable/20191021/
> +                 (V1.3 2013-09-20)}
> 
> Cut and paste oversight?

*Blush*.  Fixed in a separate commit.  Thanks for paying attention,
and this clearly is a thing that ivoatex should be automating.

When merging, I'd suggest squash-merging, as that commit will be
silly in the global history.

> +ivoatex/Makefile:
> 
> This is a good idea!

It's what the the ivoatex template has done since commit 7f384017
(Jan 2021).  I have thought about some way to implant it into the
Makefiles of the existing specs but figured anything automatic
probably is too fragile.  So, I'm adding these as specs are being
updated.

tomdonaldson commented:
I've added #32 to see if someone is inspired to add an example for this case.

msdemlei commented:
On Thu, Jan 26, 2023 at 07:06:59AM -0800, Tom wrote:
> 
> >  Cut and paste oversight?
> > *Blush*. Fixed in a separate commit. 
> 
> I'm not seeing another commit.  Was it pushed as well?  I could
> just make the change on this end but didn't want to confuse things
> if it's already in the works on your side.

Clearly not.  Sorry.

tomdonaldson reviewed (approved):
I think we're good to go here.  Thanks @msdemlei for the new content along with the maintenance updates and for starting the process towards v1.5.

msdemlei commented:
On Thu, Jan 26, 2023 at 07:24:56AM -0800, Tom wrote:
> ```suggestion
>                  (V1.4 2019-10-21)}
> ```

Ah, as I said: this needs automation.  Force-pushed a fix, please
discard your clarify-min-max-literals branch.
